experimental/dds/tree/api-report/experimental-tree.api.md

```diff
@@ -731,7 +730,7 @@ export interface ParentData {
 }
 
 // @public
-export type Payload = Serializable;
+export type Payload = any;
```

Comment:
I assume this is the same deliberate choice to leave "legacy" SharedTree alone.

Reply:
Yes.